src/js/outreach/views/opt-in_views.js

```diff
@@ -80,7 +81,12 @@ const OptInView = View.extend({
     this.setSubmitButtonState();
   },
   onChangePhone() {
-    this.model.set({ phone: this.ui.phone.val() });
+    const phone = parsePhoneNumber(this.ui.phone.val(), 'US');
+
+    this.model.set({
+      phone: phone ? phone.number : null,
```

Comment:
The parsed `phone` will either equal `undefined` or be an object like this:

```js
"phone": {
  "country": "US",
  "countryCallingCode": "1",
  "nationalNumber": "8887771234",
  "number": "+18887771234"
}
```

Reply:
interesting.. so if the phone isn't formatted correctly the submit button will be disabled.. hmm..  we _might_ need to set the value on the model and validate it to show an error on submit or something..

Reply:
Agreed, seems better than just sending a `null` value to the backend.

`parsePhoneNumber` does seem to accept almost everything though in terms of formatting. The only time I can get it to be `undefined` is if I start entering non-numerical values.

Reply:
Do we want to only allow numbers in that input?

Reply:
hmm people are going to type `-`s so it'd have to be numbers and dashes.. maybe `()`s